node_modules/oae-core/activity/css/activity.css

```diff
@@ -75,78 +85,21 @@
     margin-top: -43px;
 }
 
-/* Show all tiles. Depending on the screen resolution, a different number of tiles will be displayed.
-    - iPhone resolution: Show up to 2 tiles. If there are more tiles, the `Show all` link will be shown.
-    - Tablet resolution: Show up to 3 tiles. If there are more tiles, the `Show all` link will be shown.
-    - Desktop resolution: Show up to 4 tiles. If there are more tiles, the `Show all` link will be shown.
-    - Large screen resolution: Show up to 6 tiles. If there are more tiles, the `Show all` link will be shown. */
-
-.activity-widget .activity-preview-container ul.oae-list > li:nth-child(n+7) {
-    display: none;
-}
-
-/* Desktop resolution */
-@media (max-width: 1199px) {
-    .activity-widget .activity-preview-container ul.oae-list > li:nth-child(n+5) {
-        display: none;
-    }
-}
-
-/* Tablet resolution */
-@media (max-width: 991px) {
-    .activity-widget .activity-preview-container ul.oae-list > li:nth-child(n+4) {
-        display: none;
-    }
-}
-
-/* iPhone resolution */
-@media (max-width: 767px) {
-    .activity-widget .activity-preview-container ul.oae-list > li:nth-child(n+3) {
-        display: none;
-    }
-}
-
 .activity-widget .activity-show-all-toggle-container {
     margin-bottom: -15px;
     margin-top: 20px;
 }
 
-.activity-widget .activity-preview-container ul.oae-list[data-items="1"] + .activity-show-all-toggle-container,
-.activity-widget .activity-preview-container ul.oae-list[data-items="2"] + .activity-show-all-toggle-container {
+.activity-widget .activity-preview-container .activity-show-all-toggle-container,
+.activity-widget .activity-preview-container.activity-show-all .activity-show-all-toggle-container,
+.activity-widget .activity-preview-container.activity-show-all.activity-overflow .activity-show-all-toggle-container {
     display: none;
```

Comment:
Being able to handle these states with 3 CSS rules is actually far more simple than all the conditional checks that would have to go into the JS to do the same thing on resize and on click using `hide()`